framework/contracts/account/manager/tests/adapters.rs

```diff
@@ -379,7 +387,7 @@ fn account_install_adapter() -> AResult {
         .claim_namespace(TEST_ACCOUNT_ID, "tester".to_owned())?;
 
     let adapter = BootMockAdapter1V1::new_test(chain);
-    adapter.deploy(V1.parse().unwrap(), MockInitMsg)?;
+    adapter.deploy(V1.parse().unwrap(), MockInitMsg, DeployStrategy::Try)?;
```

Comment:
Hm, it feels a bit annoying. What if we put `DeployStrategy` into App/Adapter structs with default strategy "Try".
And in case we want to customize it make a trait, something like:
```
adapter.deploy_strategy(DeployStrategy::Force).deploy(..)
```

Reply:
I think default should actually be `Error`. Besides that adding a field to the interface struct requires way more code (because it needs to be added to the structs manually). 

I think this approach is pretty good. I had three different functions before but this way the dev really has to decide what they expect when the module is already registered.